src/containers/my-courses/my-courses-container/MyCorsesCardsList.tsx

```diff
@@ -29,18 +29,30 @@ const MyCorsesCardsList: FC<CourseListProps> = ({
   ...props
 }) => {
   const courseItems = items.map((course) => (
-    <Box key={course._id} onClick={() => onCourseSelect?.(course)}>
+    <Grid
```

Comment:
Maybe better is to use the css grid and for example if we decide to use another lib, not mui, it should work anyway

Reply:
I agree. I changed this to a simple css grid.